Makefile.bsp.in

```diff
@@ -21,14 +21,31 @@
 IMAGE_TYPES_FOR_ALL ?= msc-image-base
 IMAGE_TYPES_FOR_WAYLAND ?= msc-image-weston msc-image-qt5
 IMAGE_TYPES ?= ${IMAGE_TYPES_FOR_ALL}
+TOOLCHAIN_PATHV7 := ~/msc-ldk-toolchain-v7
+TOOLCHAIN_PATHV8 := ~/msc-ldk-toolchain-v8
+PATH := ${PATH}:${TOOLCHAIN_PATHV7}/bin:${TOOLCHAIN_PATHV8}/bin
+CURRENT_FILE := $(shell realpath Makefile.bsp.in)
+CURRENT_DIR := $(shell dirname ${CURRENT_FILE})
+BB_ENV_EXTRAWHITE="MACHINE DISTRO TCMODE TCLIBC http_proxy ftp_proxy https_proxy all_proxy ALL_PROXY no_proxy SSH_AGENT_PID SSH_AUTH_SOCK BB_SRCREV_POLICY SDKMACHINE BB_NUMBER_THREADS PARALLEL_MAKE GIT_PROXY_COMMAND GIT_PROXY_IGNORE SOCKS5_PASSWD SOCKS5_USER OEBASE META_SDK_PATH TOOLCHAIN_TYPE TOOLCHAIN_BRAND TOOLCHAIN_BASE TOOLCHAIN_PATH TOOLCHAIN_PATH_ARMV5 TOOLCHAIN_PATH_ARMV7 TOOLCHAIN_PATH_ARMV8 EXTRA_TISDK_FILES TISDK_VERSION ARAGO_BRAND ARAGO_RT_ENABLE ARAGO_SYSTEST_ENABLE ARAGO_KERNEL_SUFFIX TI_SECURE_DEV_PKG_CAT TI_SECURE_DEV_PKG_AUTO TI_SECURE_DEV_PKG_K3 ARAGO_SYSVINIT SYSFW_FILE"
```

Comment:
Is this really necessary?

Reply:
Yes Markus. This required to setup external ARM toolchain. TI SDK is build with external ARM toolchain only and not with yocto inbuilt poky arm toolchain.

https://e2e.ti.com/support/processors-group/processors/f/processors-forum/1187849/processor-sdk-am62x-processor-sdk-am62x-regarding-tool-chain-support